pyingest/parsers/gcncirc.py

```diff
@@ -28,15 +33,27 @@ def make_pubdate(self):
 
     def make_bibcode(self):
         year = self.data_dict['pubdate'][0:4]
-        bibcode = 'GCN..'
-        page = self.data_dict['page'].rjust(9,'.')
-        init = '.'
+        bibcode = 'GCN.'
+        if int(self.data_dict['page']) < 10000:
+            self.data_dict['page'] = '.' + self.data_dict['page']
```

Comment:
this would fail for volumes < 1000.  Since we may (will?) want to reprocess historical content, please update to cover all possible values for circular numbers

Reply:
This is fixed.